### ISSUE TITLE
GPU model in reference for single GPU models 

### ISSUE THREAD

**Issue by arvieFrydenlund** [tensorflow/tensor2tensor#421]
I'm wondering if transformer_big_single_gpu and transformer_base_single_gpu should fit on a Titan X or if they require a P100 to run without modification?  Thanks.

**Comment by arvieFrydenlund** [comment]
Ya, I just wanted confirmation that the default hyper-parameters are set up to use more than 12GB of memory (and I'm thinking they are just because I had to reduce the batch_size to get it to run on a Titian X).  Also is it advisable to re-search the other hyperparameters (learning rate etc.) if I do cut the batch size?  Or are the default settings robust to batch size change?

Thanks.

**Comment by lukaszkaiser** [comment]
We try to scale everything so that you can just change `batch_size=XXX` without touching anything else. I would not give a guarantee that that's optimal, but that's what we do and I get very reasonable (and sometimes SOTA) results doing just that.
